kevinb5617/ResuMake
# Start Jest Tests

# Description

The React components that makeup the project should be tested with Jest.

# Related Issue(s) and Pull Request(s)

closes #20 

# Solution

Jest was used to create unit tests for the App and Homepage components. Additionally, a new combined GitHub action was made to run the ESLint checker and the unit tests.

# Steps to Test

Assuming you have this repository has your `upstream` remote
1. `$ git fetch upstream`
2. `$ git checkout feature/start-jest-tests`
3. `$ cd app`
4. `$ npm install`
5. `$ npm start`
6. Make sure you can still see the website on `localhost:3000`
7. Stop the server with CTRL + C
8. `npm test`
9. You can run all tests with typing `a` in the test runner. All tests should pass
10. Also visit the [Actions tab](https://github.com/kevinb5617/ResuMake/actions) to see the actions being run for this branch.

# Known Issues

NONE

# Checklist
